### PR TITLE
[NT-2072] Update Manage Pledge View To Use Apollo Query

### PR DESCRIPTION
**Pull request by justinswart**
# 📲 What

Continues the migration of fetching the data for the Manage Pledge View from our legacy GraphQL queries to Apollo using fragments.

# 🤔 Why

We are using Apollo for GraphQL moving forward.

# 🛠 How

Added the remaining required fragments, adapters and tests.

# ✅ Acceptance criteria

- [ ] `Project` can be created from `ProjectFragment`.
- [ ] No regression in the behaviour of `ManagePledgeViewController`.

### CODE REVIEW

File: Library/ViewModels/ManagePledgeViewModel.swift [hunk 1]
```diff
@@ -122,10 +122,9 @@ public final class ManagePledgeViewModel:
     let graphBackingEvent = shouldFetchGraphBackingWithParam
       .map { param in param.id }
       .skipNil()
-      .map(String.init)
       .switchMap { backingId in
         AppEnvironment.current.apiService
-          .fetchManagePledgeViewBacking(query: managePledgeViewProjectBackingQuery(withBackingId: backingId))
+          .fetchManagePledgeViewBacking(id: backingId)
```
Comment (justinswart) [inline Library/ViewModels/ManagePledgeViewModel.swift:127]: Nothing like a plan coming together - this was the only change needed in `ManagePledgeViewModel`.

File: KsApi/models/graphql/adapters/Project+ProjectFragment.swift [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+private func projectStats(from projectFragment: GraphAPI.ProjectFragment) -> Project.Stats {
+  return Project.Stats(
+    backersCount: projectFragment.backersCount,
+    commentsCount: nil,
+    convertedPledgedAmount: nil,
+    currency: projectFragment.currency.rawValue,
+    currentCurrency: nil,
+    currentCurrencyRate: nil,
+    goal: projectFragment.goal?.fragments.moneyFragment.amount.flatMap(Int.init) ?? 0,
+    pledged: projectFragment.pledged.fragments.moneyFragment.amount.flatMap(Int.init) ?? 0,
```
Comment (singhhari) [inline KsApi/models/graphql/adapters/Project+ProjectFragment.swift:97]: are goal and pledged meant to be the same value?

**Review by singhhari — APPROVED** [review]
lgtm

File: KsApi/models/graphql/adapters/Project+ProjectFragment.swift (hunk 2 above)
Comment (justinswart) [reply KsApi/models/graphql/adapters/Project+ProjectFragment.swift:97]: Nope, they aren't, they're just both coalesced to `0` as a fail-safe here.

**Pull request merged**